.travis.yml

```diff
@@ -70,9 +70,8 @@ jobs:
         - if [[ "$WITH_COVERAGE" == "true" ]]; then code-climate-before-script; fi
 
       script:
-        - vendor/bin/phpunit --testsuite integration
```

Comment:
No, we don't want to collect coverage for integration tests, do we?

That was the point of #629.